Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python
+
```

Comment:
This clearly conflicts with Staff0rd's #16 (already merged).

Any chance it makes sense for us to have some sort of core docker image, and then this one and Staff0rd's can be based on that?  This one (entering in a jupyter notebook) probably isn't a good candidate for the core image, but maybe Staff0rd's is?

In any case, this branch/PR should probably be rebased on top of master as it currently stands.

Reply:
I wanted to show to @staff0rd that I was doing the same but aiming to add jupyter
So that's not an actual PR for now actually and currently in local branch I used his branch